docs/resources/live_disable_push_stream.md

```diff
@@ -0,0 +1,64 @@
+---
+subcategory: "Live"
+layout: "huaweicloud"
+page_title: "HuaweiCloud: huaweicloud_live_disable_push_stream"
+description: |-
+  Manages a disable push stream resource within HuaweiCloud.
+---
+
+# huaweicloud_live_disable_push_stream
+
+Manages a disable push stream resource within HuaweiCloud.
+
+-> Creating the resource indicates disable a push stream, deleting the resource indicates resume a push stream.
+
+## Example Usage
+
+```hcl
+variable "domain_name" {}
+variable "app_name" {}
+variable "stream_name" {}
+
+resource "huaweicloud_live_disable_push_stream" "test" {
+  domain_name = var.domain_name
+  app_name    = var.app_name
+  stream_name = var.stream_name
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the resource.
+  If omitted, the provider-level region will be used.
+  Changing this parameter will create a new resource.
+
+* `domain_name` - (Required, String, ForceNew) Specifies the ingest domain name of the disable push stream.
```

Comment:
Please consider the translation carefully, the usage of `of the disable push stream` has grammatical problems.